protected_branches.go

```diff
@@ -163,3 +165,31 @@ func (s *ProtectedBranchesService) UnprotectRepositoryBranches(pid interface{},
 
 	return s.client.Do(req, nil)
 }
+
+// UpdateRepositoryBranchesOptions represents the available
+// UpdateRepositoryBranchesOptions() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/protected_branches.html#require-code-owner-approvals-for-a-single-branch
+type UpdateRepositoryBranchesOptions struct {
+	CodeOwnerApprovalRequired *bool `url:"code_owner_approval_required,omitempty" json:"code_owner_approval_required,omitempty"`
+}
+
+// UpdateRepositoryBranches updates the given protected branch (currently only code owner approvals)
+//
+// Gitlab API docs:
+// https://docs.gitlab.com/ee/api/protected_branches.html#require-code-owner-approvals-for-a-single-branch
+func (s *ProtectedBranchesService) UpdateRepositoryBranches(pid interface{}, branch string, opt *UpdateRepositoryBranchesOptions, options ...OptionFunc) (*Response, error) {
```

Comment:
I think this name doesn't sound right. Looking at the docs it should be something like:

```suggestion
func (s *ProtectedBranchesService) RequireCodeOwnerApprovals(pid interface{}, branch string, opt *RequireCodeOwnerApprovalsOptions, options ...OptionFunc) (*Response, error) {
```